build/AzurePipelinesTemplates/ProjectReunion-RunHelixTests-Job.yml

```diff
@@ -13,21 +13,17 @@ parameters:
   # if 'useBuildOutputFromBuildId' is set, we will default to using a build from this pipeline:
   useBuildOutputFromPipeline: $(System.DefinitionId)
   matrix: 
-    Debug_x86:
```

Comment:
curious, I wonder why this was wired to Debug_x86, do we need both or one is sufficient?

Reply:
for now I think just release is fine. If we need to set up debug testing we can add it back pretty easily.